components/VotePanel/CastVoteButtons.tsx

```diff
@@ -58,6 +67,21 @@ const useCanVote = () => {
 }
 
 export const CastVoteButtons = () => {
+  const wallet = useWalletOnePointOh()
+  const connection = useWalletStore((s) => s.connection)
+  const { realm, realmInfo, config, allowDiscussion } = useRealm()
+  const { refetchProposals } = useWalletStore((s) => s.actions)
+  const isNftPlugin =
+    config?.account.communityTokenConfig.voterWeightAddin &&
+    nftPluginsPks.includes(
+      config?.account.communityTokenConfig.voterWeightAddin?.toBase58()
+    )
+  const { closeNftVotingCountingModal } = useNftProposalStore.getState()
```

Comment:
im confused about whats going on here, we are closing this modal but when does it get opened?